.github/workflows/sanitizer-build-and-test.yaml

```diff
@@ -10,7 +10,7 @@ name: Sanitizer test
       - prerelease_test
       - trigger/sanitizer
   pull_request:
-    paths: .github/workflows/sanitizer-build-and-test.yaml
+#    paths: .github/workflows/sanitizer-build-and-test.yaml
```

Comment:
I assume this is only for testing and should be reverted before merging.

Reply:
Correct.